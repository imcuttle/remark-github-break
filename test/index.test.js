/**
 * @file index
 * @author Cuttle Cong
 * @date 2018/2/24
 * @description
 */
const remark = require('remark')
const html = require('remark-html')
const breaks = require('../index')

describe('index', function () {
  test('index case 1', () => {
    remark()
      .use(html)
      .use(breaks)
      .process('aaa\\\n*b*bb  \nccc\nddd\\eee\n\\ \nfff', function (err, content) {
        expect(content.contents).toMatchSnapshot()
      })
  })
})
