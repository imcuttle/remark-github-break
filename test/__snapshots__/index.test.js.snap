// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index index case 1 1`] = `
"<p>aaa<br>
<em>b</em>bb<br>
ccc
ddd\\\\eee
\\\\
fff</p>
"
`;
